src/libaktualizr/uptane/imagerepository.cc

```diff
@@ -115,7 +115,7 @@ void ImageRepository::fetchTargets(INvStorage& storage, const IMetadataFetcher&
   verifyTargets(image_targets, false);
 
   if (local_version > remote_version) {
-    throw Uptane::SecurityException(RepositoryType::IMAGE, "Mismatched target versions");
+    throw Uptane::SecurityException(RepositoryType::IMAGE, "Rollback attempt");
```

Comment:
That looks like smth which deserves big bold red error in the log output. Is this done where this exception is caught?

Reply:
Yes, I believe so, although the real tragedy is we don't think we make this particularly obvious in the web UI.

Reply:
Hmm... I though there's a ticket for that but can't find it now.